### ISSUE TITLE
Alt Text

### ISSUE THREAD

**Issue by farishkash** [Mar50/prj-rev-bwfs-tea-cozy#2]
https://github.com/Mar50/prj-rev-bwfs-tea-cozy/blob/71cd96c6998e7105ddb2c8980fbca840ecf9c2b4/tea_cozy/index.html#L35

Each image should have a descriptive alt text.  This helps the visually impaired with their use of screen readers that read this out loud.
